docs/platforms/elixir/integrations/oban/index.mdx

```diff
@@ -28,3 +32,5 @@ config :sentry,
     ]
   ]
 ```
+
+This configuration will report failed Oban jobs to Sentry. It will also report started, completed, and failed job, alongside their duration. It will use the worker name as the `monitor_slug` of the reported cron.
```

Comment:
```suggestion

This configuration will report failed Oban jobs to Sentry. It will also report started, completed, and failed jobs, as well as their duration. It will use the worker name as the `monitor_slug` of the reported cron.
```